src/reducers/controls.ts

```diff
@@ -151,14 +152,15 @@ export interface BasicControlsState {
   zoomMin?: number
 }
 
+export interface MeasurementFilters {
+  [key: string]: Map<string, {active: boolean}>
+}
 export interface MeasurementsControlState {
   measurementsGroupBy: string | undefined,
-  measurementsDisplay: string | undefined,
+  measurementsDisplay: MeasurementsDisplay | undefined,
   measurementsShowOverallMean: boolean | undefined,
   measurementsShowThreshold: boolean | undefined,
-  measurementsFilters: {
-    [key: string]: Map<string, {active: boolean}>
-  }
+  measurementsFilters: MeasurementFilters
 }
```

Comment:
Should `MeasurementFilters` be in `measurements/types.ts` similar to `MeasurementsDisplay`?

Reply:
Hmm, `MeasurementFilters` is not used outside of the controls so I'd rather group with the control states.